content/en/docs/pipelines/sdk/component-development.md

```diff
@@ -2,7 +2,7 @@
 title = "Create Reusable Components"
 description = "A detailed tutorial on creating components that you can use in various pipelines"
 weight = 40
-                    
+
 +++
 {{% alert title="Out of date" color="warning" %}}
 This guide contains outdated information pertaining to Kubeflow 1.0. This guide
```

Comment:
```suggestion
This tutorial contains outdated information pertaining to Kubeflow 1.0. This guide
```

Reply:
Hi @Bobgy @IronPan @neuromage @Ark-kun @numerology This tutorial has the out-of-date banner fyi cc @RFMVasconcelos

Reply:
Line 12: "page" -> "tutorial"